Gemfile

```diff
@@ -0,0 +1,8 @@
+source 'https://rubygems.org'
+
+gem 'rake'
+gem 'neat'
+gem 'kramdown'
+gem 'jekyll'
+gem 'jekyll-paginate'
+gem 'jekyll-redirect-from'
```

Comment:
For some reason I have to add `json` to the list of gems for `rake dev` to work. Without it I get the following error (note that I am running an incredibly old version of Fedora on my laptop, so that might have something to do with it):

```
bundle exec jekyll serve --drafts --config _config.yml
Configuration file: _config.yml
            Source: /home/rbarlow/srobo/git/new/docs
       Destination: /home/rbarlow/srobo/git/new/docs/_site
 Incremental build: disabled. Enable with --incremental
      Generating... 
bundler: failed to load command: jekyll (/home/rbarlow/bin/jekyll)
LoadError: cannot load such file -- json
  /home/rbarlow/.gem/ruby/gems/jekyll-3.1.6/lib/jekyll/filters.rb:2:in `require'
  /home/rbarlow/.gem/ruby/gems/jekyll-3.1.6/lib/jekyll/filters.rb:2:in `<top (required)>'
  /home/rbarlow/.gem/ruby/gems/jekyll-3.1.6/lib/jekyll/renderer.rb:53:in `run'
  /home/rbarlow/.gem/ruby/gems/jekyll-3.1.6/lib/jekyll/site.rb:179:in `block in render'
  /home/rbarlow/.gem/ruby/gems/jekyll-3.1.6/lib/jekyll/site.rb:177:in `each'
  /home/rbarlow/.gem/ruby/gems/jekyll-3.1.6/lib/jekyll/site.rb:177:in `render'
  /home/rbarlow/.gem/ruby/gems/jekyll-3.1.6/lib/jekyll/site.rb:59:in `process'
  /home/rbarlow/.gem/ruby/gems/jekyll-3.1.6/lib/jekyll/command.rb:26:in `process_site'
  /home/rbarlow/.gem/ruby/gems/jekyll-3.1.6/lib/jekyll/commands/build.rb:60:in `build'
  /home/rbarlow/.gem/ruby/gems/jekyll-3.1.6/lib/jekyll/commands/build.rb:33:in `process'
  /home/rbarlow/.gem/ruby/gems/jekyll-3.1.6/lib/jekyll/commands/serve.rb:34:in `block (2 levels) in init_with_program'
  /home/rbarlow/.gem/ruby/gems/mercenary-0.3.6/lib/mercenary/command.rb:220:in `call'
  /home/rbarlow/.gem/ruby/gems/mercenary-0.3.6/lib/mercenary/command.rb:220:in `block in execute'
  /home/rbarlow/.gem/ruby/gems/mercenary-0.3.6/lib/mercenary/command.rb:220:in `each'
  /home/rbarlow/.gem/ruby/gems/mercenary-0.3.6/lib/mercenary/command.rb:220:in `execute'
  /home/rbarlow/.gem/ruby/gems/mercenary-0.3.6/lib/mercenary/program.rb:42:in `go'
  /home/rbarlow/.gem/ruby/gems/mercenary-0.3.6/lib/mercenary.rb:19:in `program'
  /home/rbarlow/.gem/ruby/gems/jekyll-3.1.6/bin/jekyll:13:in `<top (required)>'
  /home/rbarlow/bin/jekyll:23:in `load'
  /home/rbarlow/bin/jekyll:23:in `<top (required)>'
rake aborted!
Command failed with status (1): [bundle exec jekyll serve --drafts --config...]
/home/rbarlow/srobo/git/new/docs/Rakefile:2:in `block in <top (required)>'
/home/rbarlow/.gem/ruby/gems/rake-11.3.0/exe/rake:27:in `<top (required)>'
Tasks: TOP => dev
(See full trace by running task with --trace)
```